ui/app/templates/vault/cluster/access/leases/list.hbs

```diff
@@ -57,28 +57,26 @@
       <div class="control">
         {{#if (and this.capabilities.forceRevokePrefix.canUpdate (not this.confirmingRevoke))}}
           <ConfirmAction
-            @buttonClasses="toolbar-link"
-            @confirmTitle="Disable this?"
-            @confirmMessage={{concat "All leases under this one will also be removed and disregard any errors encountered."}}
-            @confirmButtonText="Revoke"
+            @buttonText="Force revoke prefix"
+            class="toolbar-button"
+            @buttonColor="secondary"
+            @confirmTitle="Revoke this?"
+            @confirmMessage="All leases under this one will also be removed and disregard any errors encountered."
```

Comment:
I know it was already this way but the phrasing seems strange. To me something like this makes more sense:
```suggestion
            @confirmMessage="All leases under this one will also be removed and any errors encountered will be disregarded."
```